pkg/slack/events/messages/message_handler.go

```diff
@@ -84,26 +84,51 @@ func postResponse(client *slack.Client, event *slackevents.MessageEvent, respons
 }
 
 func help(client *slack.Client, event *slackevents.MessageEvent, botCommands []parser.BotCommand) {
-	helpMessage := " "
-	helpMessage += "help" + " - " + fmt.Sprintf("_%s_", "help") + "\n"
+	helpMessage := ""
 	for _, command := range botCommands {
 		tokens := command.Tokenize()
+
+		// # <command>
+		helpMessage += "# "
```

Comment:
Unfortunately, Slack's `mrkdwn` language does not fully support everything that traditional Markdown does, like Headings:
https://api.slack.com/reference/surfaces/formatting#basics

Reply:
Here is how your current changes render:
![image](https://github.com/openshift/ci-chat-bot/assets/43015045/d16d5013-1a67-4251-b84b-d3825fdf4f11)

Reply:
Damn, I really should have sent some messages to myself/RTM rather than relying on third-party blogs. Let me think on how to rework this further...